
We decided exposing the (maybe) slightly personal/secret .plan is a bad idea.

  - Rene Rebe <rene@exactcode.de>

diff -ur xlockmore-5.14.1/xlock/xlock.c xlockmore-5.14.1-fixed/xlock/xlock.c
--- xlockmore-5.14.1/xlock/xlock.c	2004-11-09 23:07:06.000000000 +0100
+++ xlockmore-5.14.1-fixed/xlock/xlock.c	2005-01-11 16:00:32.543487472 +0100
@@ -2963,6 +2963,8 @@
 	(void) sprintf(buffer, "%s/%s", home, ".xlocktext");
 #endif
 	planf = my_fopen(buffer, "r");
+
+#ifdef	PLAN_WANTED	/* disable .plan exposing for T2 -ReneR */
 	if (planf == NULL) {
 #ifdef VMS
 		(void) sprintf(buffer, "%s%s", home, ".plan");
@@ -2971,6 +2973,7 @@
 #endif
 		planf = my_fopen(buffer, "r");
 	}
+#endif /* PLAN_WANTED */
 	if (planf == NULL) {
 #ifndef VMS
 		(void) sprintf(buffer, "%s/%s", home, ".signature");
diff -ur xlockmore-5.14.1/xlock/xlock.man xlockmore-5.14.1-fixed/xlock/xlock.man
--- xlockmore-5.14.1/xlock/xlock.man	2004-11-02 02:23:21.000000000 +0100
+++ xlockmore-5.14.1-fixed/xlock/xlock.man	2005-01-11 16:00:50.570746912 +0100
@@ -291,7 +291,7 @@
 
 In the lower part of the password screen a text is displayed.  This
 message is taken from the first file of the following that exists:
-$HOME/.xlocktext, $HOME/.plan, or $HOME/.signature.
+$HOME/.xlocktext or $HOME/.signature.
 
 On systems which support new BSD style authentication, the password
 may be prefixed by an authentication style followed by a colon
